### PR TITLE
serverProfile: fix crash with lurked guilds

### PR DESCRIPTION
**Pull request by aamiaa**
This fixes a crash when using the plugin on a lurked guild.

### CODE REVIEW

File: src/plugins/serverProfile/GuildProfileModal.tsx [hunk 1]
```diff
@@ -170,7 +170,7 @@ function ServerInfoTab({ guild }: GuildProps) {
     const Fields = {
         "Server Owner": owner ? Owner(guild.id, owner) : "Loading...",
         "Created At": renderTimestamp(SnowflakeUtils.extractTimestamp(guild.id)),
-        "Joined At": renderTimestamp(guild.joinedAt.getTime()),
+        "Joined At": guild.joinedAt ? renderTimestamp(guild.joinedAt.getTime()) : "-", // Lurked guild
```
Comment (TheKodeToad) [inline src/plugins/serverProfile/GuildProfileModal.tsx:173]:
```suggestion
        "Joined At": guild.joinedAt ? renderTimestamp(guild.joinedAt.getTime()) : "-", // Not available in lurked guild
```
every little helps :-)

**Comment by Vendicated** [comment]
thanks

**Comment by aamiaa** [comment]
Seems like this never actually got merged, somehow: https://github.com/Vendicated/Vencord/blob/main/src/plugins/serverProfile/GuildProfileModal.tsx#L173

**Comment by Vendicated** [comment]
sorry i was hungry and the commit looked extra delicious

(something went wrong in the mirror pipeline and it swallowed some commits, i added them back manually but apparently missed yours)

**Comment by Vendicated** [comment]
https://github.com/Vendicated/Vencord/commit/c33d59b45dc2e63ff80ce02287415f8973d90841

**Pull request merged**